src/Typography/Typography.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React, { ElementType, FC } from 'react';
+
+import styles from './Typography.module.css';
+
+import { TypographyProps } from './Typography.types';
+
+const TYPOGRAPHY_VARIANTS: { [variant: string]: ElementType } = {
+  hero: 'h1',
+  h1: 'h1',
+  h2: 'h2',
+  h3: 'h3',
+  h4: 'h4',
+  h5: 'h5',
+  h6: 'h6',
+  subtitle: 'h6',
+  body1: 'p',
+  body2: 'p',
+  caption: 'p',
+  disclaimer: 'p',
+};
+
+/** The Typography component makes it easy to apply a default set of font weights and sizes in the application. */
+const Typography: FC<TypographyProps> = ({
+  className,
+  variant = 'body1',
+  style,
```

Comment:
why we did not declare this prop in the interface and it is purpose ?

Reply:
We might not need it at all if we fixed that [issue](https://github.com/m3ntorship/m3ntorship-ui/pull/28#discussion_r905184549) first